WordPress/Classes/ViewRelated/Post/PostCardStatusViewModel.swift

```diff
@@ -150,7 +150,9 @@ class PostCardStatusViewModel: NSObject {
                 buttons.append(.cancelAutoUpload)
             }
 
-            if canPublish {
+            if post.autoUploadAttemptsCount.intValue >= PostAutoUploadInteractor.maxNumberOfAttempts {
+                buttons.append(.retry)
+            } else if canPublish {
```

Comment:
⚠️  I don't think that the Publish and Retry buttons are mutually exclusive. Consider this scenario when uploading a draft:

Attempt | Buttons Available 
--------|-------
0        |       Edit, Publish, Trash
1 | Edit, Publish, Trash
2 | Edit, Publish, Trash
3 | Edit, Retry, Trash

When attempt number 3 happens, there is no way to Publish the draft anymore. The buttons should be: Edit, Retry, Publish, and Trash. The Publish and Trash will probably end up being underneath _More_.

Reply:
Done in 9dbfee3b2805574bed17a2f8391e7782e3d22ccf